api/src/dtos/fantasy-teams.dto.ts

```diff
@@ -0,0 +1,42 @@
+import { IsString, IsNotEmpty, MinLength, MaxLength, IsNumber } from 'class-validator';
+
+export class CreateFantasyTeamDto {
+  @IsString()
+  @IsNotEmpty()
+  @MinLength(1)
+  @MaxLength(40)
+  public team_name: string;
+
+  @IsNumber()
+  public owner: number;
+
+  @IsString()
+  @IsNotEmpty()
+  @MinLength(1)
+  public logo: string;
+
+  public points: number;
+
+  public qb: number;
```

Comment:
For the positions, maybe you can use the Min class validator and set the minimum value to 0, because id will never be negative